pytest_logger/plugin.py

```diff
@@ -13,6 +14,28 @@
 else:
     string_type = str
 
+os_symlink = getattr(os, "symlink", None)
```

Comment:
Another option is to isolate the change into function that uses it (only one in this file). If python supports symlink (any extra include or natively), then this code is not used.